### DISCUSSION TITLE
How can I get the rank of a listing?

### DISCUSSION THREAD

**Discussion by urikanonov** [etsy/open-api#1157] [Ideas]
Hi, 
In Etsy each listing has a custom 'rank' / 'order' in which it is displayed in the shop.
I saw that there is the 'featured_rank' (order in featured section) in the API listing object but not the overall rank.

Is there any way to get the shop-wide rank?

If not, how can this be raised as a feature request?

Thank you!
Uri

**Comment by Gareth064** [comment]
Unfortunately there isnt

**Reply by urikanonov** [reply]
How can I request to add it?

**Reply by Gareth064** [reply]
Email developer@etsy.com to see what they say.

**Comment by simonj** [comment]
What you mean of rank. what place it is in the search?

**Reply by urikanonov** [reply]
I mean that every shop owner can set a custom order for the listings in the store via drag and drop and I wpuld like to get the index of the listing in that custom ordering.

**Comment by urikanonov** [comment]
Anyone from Etsy here can provide any feedback?

**Reply by Gareth064** [reply]
If you can't find it in the API docs then it's not possible.

Etsy doesn't talk to people on this GitHub so if you want them to comment on the topic, you need to email them.
